lib/Doctrine/ORM/Tools/Console/Command/ConvertDoctrine1SchemaCommand.php

```diff
@@ -180,6 +180,7 @@ public function convertDoctrine1Schema(array $fromPaths, $destPath, $toType, $nu
                 );
             }
         }
+        unset($dirName);
```

Comment:
It's more tricky: https://3v4l.org/QKOKH - here unset should clean the scope after foreach.